package.json

```diff
@@ -27,7 +29,8 @@
     "lint-fix": "eslint . --fix",
     "test": "cross-env NODE_ENV=test ava",
     "coverage": "cross-env NODE_ENV=test nyc ava",
-    "report-coverage": "nyc report --reporter=text-lcov | coveralls"
+    "report-coverage": "nyc report --reporter=text-lcov | coveralls",
+    "version": "babel-node build/version.js"
```

Comment:
Ah, really? Somebody needs to tell npm to update [their docs](https://docs.npmjs.com/misc/scripts#description):

> preversion, version: Run BEFORE bump the package version

:stuck_out_tongue_winking_eye: